fms/modules/attention.py

```diff
@@ -112,6 +112,26 @@ def reset_parameters(self):
                 if self.use_bias:
                     m.bias.data.zero_()
 
+    def fuse(self):
```

Comment:
unfortunate if we can't re-use this in the adapter somehow. Do we have any use-case for the fuse operation?

Reply:
Removed fuse for now as there is not a use-case yet